content/hardware/07.opta/opta-family/opta/tutorials/01.user-manual/content.md

```diff
@@ -21,12 +21,14 @@ software:
 
 This user manual will provide a comprehensive overview of Opta™, covering its major hardware and software elements. With this user manual, you will learn how to set up, configure, and use all the main features of an Opta™ device. 
 
+![ ](hero-banner.png)

```

Comment:
Almost forgot this one. I see the banner image was placed under the docs "Abstract" to space further from the title. Thus, I agree with the placement too. Just wanted to comment if someone asks why it's placed in this manner,